cwl/tools/emerald/emerald.cwl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: CommandLineTool
+
+label: "emeraldbgc"
```

Comment:
I guess emerald is an old name of tool. Maybe it makes sense to rename a folder and cwl file to sanntis?

Reply:
I'll rename in another branch